tests/pipelines/test_byoc.py

```diff
@@ -124,12 +126,41 @@ def test_temporal_byoc(config_and_stats_paths, preparation_config, config, confi
     assert check_request.method == "GET"
 
     timestamps = [
```

Comment:
why did these timestamps change?

Reply:
because the `data_2019` was thrown out when we reduced download tests, so it's now using another downloaded data set

Reply:
it's not in the pipeline chain and thus never got tested in CI. Kinda concerning that we never ran it locally :upside_down_face: